ansible_files/aws_ec2_region_cleanup.yml

```diff
@@ -0,0 +1,148 @@
+- hosts: localhost
+  gather_facts: false
+  vars:
+    aws_region: "{{ lookup('env', 'AWS_REGION') | default('us-east-1') }}"
+    leak_threshold_seconds: "{{ lookup('env', 'LEAK_THRESHOLD_SECONDS') | default('21600') | int }}" # Default to 6 hours (21600s)
+    slack_token: "{{ lookup('env', 'SLACK_TOKEN') | default('') }}"
+    slack_channel: "{{ lookup('env', 'SLACK_CHANNEL') | default('#assisted-deployment-ci') }}"
+    slack_mention: "{{ lookup('env', 'SLACK_MENTION') | default('') }}"
+    leaked_instances: []
+    terminate_leaked_instances_flag: "{{ lookup('env', 'TERMINATE_LEAKED_INSTANCES_FLAG') | default(false) | bool }}"
+
+  tasks:
+    - name: Preliminary check for Slack Token
+      ansible.builtin.assert:
+        that:
+          - slack_token | trim != ""
+        fail_msg: "Slack Token (SLACK_TOKEN environment variable) is not configured or is empty. Please set it."
+      run_once: true
+      when: slack_token | trim == ""
+
+    - name: Gather standard Ansible facts
+      ansible.builtin.setup:
+      delegate_to: localhost
```

Comment:
doesn't look you rely on ansible fact, I think you can remove this block

Reply:
I need it for the time, but I just marked `gather_facts: true` at the beginning